.ci/release.sh

```diff
@@ -5,6 +5,29 @@ set -eou pipefail
 DOWNLOAD_BASE="$1"
 VERSION="$2"
 
+remove_homebrew() {
+  rm -rf "/tmp/b"
+}
+
+install_homebrew() {
+  # do an alternative install of homebrew to ensure we have access to
+  # perform brew operations
+  # https://docs.brew.sh/Installation#untar-anywhere
+  # note that homebrew needs to be installed somewhere with a shortish
+  # path to avoid issues relinking dynamic libraries
+  # https://github.com/Homebrew/brew/issues/4979
+  remove_homebrew
+  git clone https://github.com/Homebrew/brew.git "/tmp/b"
+  #shellcheck disable=SC2046
+  eval $("/tmp/b/bin/brew" shellenv)
```

Comment:
It would make sense to have a global variable for `/tmp/b` as it's used a few time and the folder name isn't really explicit.

Reply:
Yes, done in https://github.com/elastic/homebrew-tap/pull/78/commits/2ab3a10d571c7105c37157df21e808f45d9e399e